eth_tester/utils/backend_testing.py

```diff
@@ -489,9 +490,10 @@ def test_call_add(self, eth_tester):
         result = _decode_math_result('add', raw_result)
         assert result == (20,)
 
+    @pytest.mark.skipif(is_pyethereum16_available(), reason="v1.6 not supported")
```

Comment:
Hrm, instead of skipping this here, I think it needs to be skippend in the pyethereum16 specific subclass of this testing class.  Otherwise, this test will get skipped if pyethereum16 is installed at all, even if I'm testing the `py-evm` backend.